swarm/tools/runs_gc.py

```
@@ -74,15 +75,25 @@ def age_days(self) -> float:
-        for entry in path.rglob("*"):
-            if entry.is_file():
-                try:
-                    total += entry.stat().st_size
-                except OSError:
-                    pass
-    except OSError:
-        pass
+
+    # Performance optimization: Replace `path.rglob` with `os.scandir`
+    # to significantly reduce overhead from Path object creation and `stat()` calls.
+    def _scan_dir(dir_path: str):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_scan_dir`

Add return type annotation: `None`

(ANN202)